scripts/refresh-assets.ts

```diff
@@ -25,18 +25,21 @@ const FONT_MAPPING: Record<string, string> = {
 
 const ASSETS_FOLDER_PATH = join(REPO_ROOT, 'assets')
 
-const collectedAssetUrls: {
-	fonts: Record<string, string>
-	icons: Record<string, string>
-	translations: Record<string, string>
-	embedIcons: Record<string, string>
-} = {
+const collectedAssetUrls: Record<
+	'fonts' | 'icons' | 'translations' | 'embedIcons',
+	Record<string, { file: string; hash?: string }>
+> = {
 	fonts: {},
 	icons: {},
 	translations: {},
 	embedIcons: {},
 }
 
+const mergedIconName = '0_merged.svg'
+const mergedIconHeader =
+	'<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="none">'
```

Comment:
does the width/height/fill here do anything?

Reply:
yes, they determine the size of everything in here and how fills work. this is what svgo outputs for all of our icons, and we need to make sure they all match. I've added a comment to clarify.